src/main/i18n/templates/strings.properties

```diff
@@ -892,3 +892,9 @@ observer.tools.flyspeed.hyperspeed = Hyperspeed
 
 observer.tools.visibility.shown = Shown
 observer.tools.visibility.hidden = Hidden
+
+discord.notfound = Discord user {0} was not found on our server.
+discord.verify = Link your accounts by running this command in-game: ```{0}```
```

Comment:
`{0} requested to link their Minecraft account to your Discord. If this is you, please run the following command in-game: {1}`